### PR TITLE
Add parse ped script

### PR DESCRIPTION
**Pull request by vivbak**
A simple script to parse a ped file into SM. Created to by-pass permissions issues.

### CODE REVIEW

File: scripts/parse_ped.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    bucket = client.get_bucket(bucket_name)
+    blob = bucket.blob(ped_file_path)
+    ped_file = blob.download_as_bytes()
+
+    tmp_ped_tsv = 'tmp_ped.tsv'
+    # Work-around as import_pedigree takes a file.
+    with open(tmp_ped_tsv, 'wb') as tmp_ped:
+        tmp_ped.write(ped_file)
+
+    with open(tmp_ped_tsv) as ped_file:
```
Comment (vladsaveliev) [inline scripts/parse_ped.py:29]: I'm a bit lost here. You reuse `ped_file` variables for 2 unrelated purposes?

File: scripts/parse_ped.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+@click.option('--bucket', 'bucket_name')
+@click.option('--project', 'project')
+def main(ped_file_path: str, bucket_name: str, project: str):
+    """Pull ped file from GCS and import to SM"""
+
+    fapi = FamilyApi()
+    client = storage.Client()
+    bucket = client.get_bucket(bucket_name)
+    blob = bucket.blob(ped_file_path)
+    ped_file = blob.download_as_bytes()
```
Comment (vladsaveliev) [inline scripts/parse_ped.py:22]: Just a note, you can use [AnyPath](https://github.com/drivendataorg/cloudpathlib) to read cloud object contents:

```
path = AnyPath(ped_file_path)
with path.open() as fh:
    contents = fh.read()
```

File: scripts/parse_ped.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+    tmp_ped_tsv = 'tmp_ped.tsv'
+    # Work-around as import_pedigree takes a file.
+    with open(tmp_ped_tsv, 'wb') as tmp_ped:
+        tmp_ped.write(ped_file)
+
+    with open(tmp_ped_tsv) as ped_file:
+        fapi.import_pedigree(
+            file=ped_file,
```
Comment (vladsaveliev) [inline scripts/parse_ped.py:31]: What exactly does it expect as `file` here?

Comment (vladsaveliev) [reply scripts/parse_ped.py:31]: Ah, a file handler. 

That's how Michael seem to be using it:
https://github.com/populationgenomics/sample-metadata/blob/130c1dc20ec3e4c06b71ad88dd4df2a9fbcba7d6/test/data/generate_data.py#L42-L45

So I'd do the following:

```py
fapi = FamilyApi()
with AnyPath(ped_file_path).open() as fh:
    fapi.import_pedigree(
            file=ped_file,
            has_header=True,
            project=project,
            create_missing_participants=True,
        )
```

**Review by vladsaveliev — APPROVED** [review]
Feel free to merge if it works, but left a few comments just for info

**Comment by vivbak** [comment]
Thanks very much @vladsaveliev, have cleaned it up accordingly!

**Pull request merged**